apps/ledger-live-desktop/src/renderer/screens/dashboard/index.tsx

```diff
@@ -99,14 +101,20 @@ export default function DashboardPage() {
           <EmptyStateInstalledApps />
         ) : totalAccounts > 0 ? (
           <>
-            <BalanceSummary
-              counterValue={counterValue}
-              chartColor={colors.wallet}
-              range={selectedTimeRange}
-            />
-
-            {marketPerformanceEnabled ? <MarketPerformance /> : null}
-
+            <Flex>
+              <Box flex={2}>
+                <BalanceSummary
+                  counterValue={counterValue}
+                  chartColor={colors.wallet}
+                  range={selectedTimeRange}
+                />
+              </Box>
+              {marketPerformanceEnabled ? (
+                <Box flex={1} ml={6}>
+                  <MarketPerformanceWidget variant={marketPerformanceVariant} />
+                </Box>
+              ) : null}
+            </Flex>
```

Comment:
Can you make sure we have the same margin used here and for the cards above, please?
<img width="998" alt="image" src="https://github.com/LedgerHQ/ledger-live/assets/31533861/2a157b0d-37ff-4bc0-becd-835807eb6cb0">

Reply:
+ we need to manage the overflow inside the Graph card:
<img width="476" alt="image" src="https://github.com/LedgerHQ/ledger-live/assets/31533861/3e10dd5a-d96b-4e00-929f-ff29ccf47e11">

Reply:
Don't have this second case on my own 🤔 Which resolution is this?

Reply:
Minimal width + High variation
<img width="718" alt="image" src="https://github.com/LedgerHQ/ledger-live/assets/31533861/0cad3781-a373-49a2-9565-1181781548e9">